modules/tensor_mechanics/src/materials/ADComputeInstantaneousThermalExpansionFunctionEigenstrain.C

```diff
@@ -55,17 +55,18 @@ ADComputeInstantaneousThermalExpansionFunctionEigenstrain::computeThermalStrain(
     _step_one = false;
 
   const Real & current_temp = MetaPhysicL::raw_value(_temperature[_qp]);
```

Comment:
yeah, remove raw_value here. That's dropping derivatives

Reply:
No, this isn't used in derivatives, temp is. (Side note, I couldn't figure out how to get derivatives from `_temperature[_qp]` ...`_temperature[_qp].derivatives()`?)

Reply:
Just to be clear, the two things (raw_value and delta_T) are intimately connected...

Reply:
Yeah I think I fixed it now, take a look at the updated code.